src/mapeo-manager.js

```diff
@@ -456,4 +474,36 @@ export class MapeoManager {
   get invite() {
     return this.#invite
   }
+
+  /**
+   * @returns {Promise<PublicPeerInfo[]>}
+   */
+  async listLocalPeers() {
+    return omitPeerProtomux(this.#localPeers.peers)
+  }
```

Comment:
it doesn't feel like this strictly needs to be async but i have this sentiment elsewhere in the codebase, so not worth changing 😄